.travis.yml

```diff
@@ -0,0 +1,5 @@
+language: node_js
+node_js:
+  - "iojs"
```

Comment:
We're not bothered about the latest version of `iojs` since it's no longer supported. Please change this for `node` which will be the latest version of node.